public/admin/templates/football.tmpl.html

```diff
@@ -38,6 +38,7 @@ <h2 class="ui header">{{clock}}</h2>
                 </div>
 
                 <div class="ui button" ng-click="resetClock()">Reset</div>
+                <checkbox ng-model="football.showet" size="large" type="toggle">Extra Time</checkbox>
```

Comment:
Can the attribute on football be `show_extratime` or something instead of `showet` so its more descriptive.